schema/definitions.json

```diff
@@ -167,22 +167,21 @@
       "type": "array",
       "items": {
         "type": "string",
-        "enum": ["browser"]
+        "enum": ["browser", "unofficial"]
```

Comment:
I don't think we should re-use categories for this - this is more of a status thing than a category; if we were to start other categories (e.g. linked data, verifiable credentials, mini-apps), they could also have unofficial drafts that we may want to handle differently.

Reply:
My problem is that we'll want to override what we want to consider "unofficial" in some cases, and I don't think we should override the status of the spec. How would you like to do that?

Reply:
I think we want a separate entry that can be used to filter whether / where the extracted data is taken into account; if we focus on status, it could be something labeled e.g. "standing" (?); or maybe we're really talking about implementation level, in which case the label could reflect that?

Reply:
What we want to achieve strikes me as ill-defined. I see how we could call specs we'd want to exclude from browser-specs ("unofficial", "idea", "proposal") but I don't know how to call specs we want to keep ("official" feels too strong, "implementers expressed interest" somewhat arbitrary). Hence the use of "categories" which, I agree, is not ideal, but allows to use only one term and remain implicit on what it means when this term is missing ;)

We could perhaps override the "status" property after all. That is, we could say that we consider font-metrics-api-1 and web-animations-2 to be proper drafts. One example of spec for which I find this approach somewhat clunky would be [Web Monetization](https://webmonetization.org/specification.html), which is currently on our monitoring list. Status is CG draft. The spec is explicit that it is "work in progress" but, if we add it to the list, calling it "Unofficial Proposal Draft" feels a tad too strong.

Or we create a `standing` property whose meaning remains a bit fuzzy and whose value can either be:
- `good`, meaning spec has some level of standing.
- `pending` or `ongoing`, meaning spec should still be regarded as unofficial
How does that look?

Reply:
I'm fine with your `standing` proposal; the more I think about it, the more this seems to be about implementation interest, but my guess is that we can start with `standing` and make it evolve when things get less fuzzy as we start using it.

Reply:
OK, done in #813!